airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/CreditsUsage.module.scss

```diff
@@ -0,0 +1,16 @@
+@use "scss/variables";
+
+.empty {
+  height: 100%;
+  padding-bottom: variables.$spacing-xl;
+}
+
+.chartWrapper {
+  width: 100%;
+  height: 260px;
```

Comment:
we don't currently have any standardized height variables, but i wonder if we should.